app/code/core/Mage/AdminNotification/Helper/Data.php

```diff
@@ -90,44 +74,4 @@ public function getUnreadNoticeCount($severity)
         }
         return isset($this->_unreadNoticeCounts[$severity]) ? $this->_unreadNoticeCounts[$severity] : 0;
     }
-
-    /**
-     * Retrieve Widget Popup Notification Object URL
-     *
-     * @param bool $withExt
-     * @return string
-     */
-    public function getPopupObjectUrl($withExt = false)
```

Comment:
This method is used by `Mage_Adminhtml_Block_Notification_Window::getObjectUrl()`. Not sure how.

Reply:
You're right as always, I was looking into removing the "window" class altogether (I wrote it in the original issue) then I thought it was too much and this part slipped from me.

Now I'm not sure what to do the `getObjectUrl` method looks for a "swf", which now nobody can run so... it's more useless than ever... should I remove the `getObjectUrl` method altogether? it seems nobody is calling it.

Reply:
If you drop them, @Flyingmana will says that's a BC.
Perhaps drop content and `return null or false` and mark functions as deprecated for 19.x, and drop functions for 20.x.

Reply:
I've restored the methods, returning nothing, adding a deprecation notice. One method was deprecated since magento 1.4 thus I've removed it completely ;-)